src/current/_includes/molt/replicator-metrics.md

```diff
@@ -0,0 +1,27 @@
+### Replicator metrics
```

Comment:
@crash-overdrive just wanted to give you a heads up that we already have a base `Replicator metrics` section that Ryan added. 

Your goal for next sprint is to flesh this out with the newest metrics details and updates to the dashboard.

Reply:
@taroface we can make the changes I recommended for now since it's a good starting point for people to get insights into observability. I'll have @crash-overdrive clean up this section and give you content for refining this doc for the upcoming release.